UI/frontend-plugins/frontend-tools/data/scripts/instant-replay.lua

```diff
@@ -22,27 +25,45 @@ function try_play()
 
 	obs.obs_output_release(replay_buffer)
 
+	if path == last_replay then
+		path = nil
+	end
+
 	-- If the path is valid and the source exists, update it with the
 	-- replay file to play back the replay.  Otherwise, stop attempting to
-	-- replay after 10 seconds
+	-- replay after number of attempts exceeds max_attempts
```

Comment:
This comment is incorrect.  The code will stop attempting to replay after the number of attempts _reaches_ max_attempts, not exceeds.  That's the difference between `>=` and `>`.